.github/workflows/image.yml

```diff
@@ -107,118 +112,38 @@ jobs:
           name: symbolicator-debug@${{ matrix.arch }}
           path: /tmp/debug-info/*
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
       - name: Prepare Docker Context
         run: |
           mkdir docker-ctx
           cp Dockerfile docker-ctx/
           mv symbolicator docker-ctx/
 
-      - name: Build Image
-        uses: docker/build-push-action@v6
+      - name: Build and push ghcr image
+        uses: getsentry/action-build-and-push-images@a53f146fc1ea3cb404f2dcf7378f5b60dd98d3ca
         with:
-          context: docker-ctx/
+          image_name: 'symbolicator'
           platforms: linux/${{ matrix.arch }}
-          tags: symbolicator-${{ matrix.arch }}
-          outputs: type=docker,dest=/tmp/symbolicator-${{ matrix.arch }}.tar
-          push: false
-
-      - name: Upload Image
-        uses: actions/upload-artifact@v4
+          dockerfile_path: './Dockerfile'
+          build_context: './docker-ctx'
+          ghcr: true
+          tag_nightly: false  # todo true
+          tag_latest: false
+
+      - name: Build and push production image
+        if: github.ref_name == github.event.repository.default_branch
+        uses: getsentry/action-build-and-push-images@a53f146fc1ea3cb404f2dcf7378f5b60dd98d3ca
         with:
-          name: symbolicator-image@${{ matrix.arch }}
-          path: /tmp/symbolicator-${{ matrix.arch }}.tar
-
-  assemble-ghcr:
-    needs: [build-setup, build-image]
-    if: "needs.build-setup.outputs.full_ci == 'true'"
```

Comment:
this is now done on every commit since e2e now needs ghcr.io/getsentry/symbolicator:${{ github.sha }} because it can't download symbolicator-amd64.tar anymore